### PR TITLE
Verification for plots targeted in Correlates of Risk Report

### PR DESCRIPTION
**Pull request by dilu963**

### CODE REVIEW

File: cor_graphical/verification/verification_report.Rmd [hunk 1]
```diff
@@ -0,0 +1,453 @@
...
+
+clean_comparison_pass_boxplot <- all(sapply(data_clean_comparison_boxplot, `[[`, "equal"))
+
+clean_comparison_pass_lineplot <- all(sapply(data_clean_comparison_lineplot, `[[`, "equal"))
+
+```
+
+The violin plot, box plot and line plot generated by the tester were visually compared with the outputs generated by original programmer. 
+
+Key statistics(min, 25th percentile, median, 75th percentile, max) shown in the box plot calculated by tester were `r ifelse(clean_comparison_pass, "equivalent to","different from")` the numbers provided by original programmer, and `r ifelse(clean_comparison_pass, "passes","fails")` verification.
```
Comment (thebioengineer) [inline cor_graphical/verification/verification_report.Rmd:273]: Uses wrong object for comparison - update to `clean_comparison_pass_boxplot`

File: cor_graphical/verification/verification_report.Rmd [hunk 2]
```diff
@@ -0,0 +1,453 @@
...
+
+clean_comparison_pass_boxplot <- all(sapply(data_clean_comparison_boxplot, `[[`, "equal"))
+
+clean_comparison_pass_lineplot <- all(sapply(data_clean_comparison_lineplot, `[[`, "equal"))
+
+```
+
+The violin plot, box plot and line plot generated by the tester were visually compared with the outputs generated by original programmer. 
+
+Key statistics(min, 25th percentile, median, 75th percentile, max) shown in the box plot calculated by tester were `r ifelse(clean_comparison_pass, "equivalent to","different from")` the statistics provided by original programmer, and `r ifelse(clean_comparison_pass, "passes","fails")` verification.
```
Comment (thebioengineer) [inline cor_graphical/verification/verification_report.Rmd:342]: Uses wrong object for comparison - update to `clean_comparison_pass_boxplot`

File: cor_graphical/verification/verification_report.Rmd [hunk 3]
```diff
@@ -0,0 +1,453 @@
...
+clean_comparison_pass_boxplot <- all(sapply(data_clean_comparison_boxplot, `[[`, "equal"))
+
+clean_comparison_pass_lineplot <- all(sapply(data_clean_comparison_lineplot, `[[`, "equal"))
+
+```
+
+
+The violin plot, box plot and line plot generated by the tester were visually compared with the outputs generated by original programmer. 
+
+Key statistics(min, 25th percentile, median, 75th percentile, max) shown in the box plot calculated by tester were `r ifelse(clean_comparison_pass, "equivalent to","different from")` the statistics provided by original programmer, and `r ifelse(clean_comparison_pass, "passes","fails")` verification.
```
Comment (thebioengineer) [inline cor_graphical/verification/verification_report.Rmd:408]: Uses wrong object for comparison - update to `clean_comparison_pass_boxplot`

File: cor_graphical/verification/verification_report.Rmd [hunk 4]
```diff
@@ -0,0 +1,453 @@
...
+```
+
+## Description
+
+This is the verification report for the `cor_graphical` folder of the `correlates_reporting` project for CoVPN. 
+
+In the verification process, the outputs of `cor_graphical_verification_code.R` were visually compared against the plots listed in the file `cor_graphical/verification/verification_plan.pdf`. The values presented in the original plots were also generated independently by the tester and compared against the outputs generated by the original programmer.
+
+The files `cor_graphical/verification/input/longer_cor_data_plot1.csv`, and \newline `cor_graphical/verification/input/plot.25sample1.csv`, 
+ were provided by the original programmer to the tester for verification purposes.
```
Comment (thebioengineer) [inline cor_graphical/verification/verification_report.Rmd:144]: Include the files indicating the sampling subsets I think

**Review by thebioengineer — CHANGES_REQUESTED** [review]
Thanks for the submission, @dilu963. Noticed some bugs in the report, please fix those and I think we are good to go!

File: cor_graphical/verification/verification_report.Rmd (hunk 1 above)
Comment (dilu963) [reply cor_graphical/verification/verification_report.Rmd]: Bug has been fixed.

File: cor_graphical/verification/verification_report.Rmd (hunk 2 above)
Comment (dilu963) [reply cor_graphical/verification/verification_report.Rmd]: Bug has been fixed

File: cor_graphical/verification/verification_report.Rmd (hunk 3 above)
Comment (dilu963) [reply cor_graphical/verification/verification_report.Rmd]: Bug has been fixed

File: cor_graphical/verification/verification_report.Rmd (hunk 4 above)
Comment (dilu963) [reply cor_graphical/verification/verification_report.Rmd:144]: plot.25sample1.csv is the sampling subset. It includes all the data points that were selected for line plot plotting.

**Review by thebioengineer — APPROVED** [review]
From what I can tell, everything looks good. Thanks @dilu963 !

**Pull request merged**